server/manager/src/main/java/org/apache/accumulo/manager/tableOps/merge/MergeTablets.java

```diff
@@ -233,6 +233,9 @@ static void validateTablet(TabletMetadata tabletMeta, String fateStr, TabletOper
     Preconditions.checkState(expectedTableId.equals(tabletMeta.getTableId()),
         "%s tablet %s has unexpected table id %s expected %s", fateStr, tabletMeta.getExtent(),
         tabletMeta.getTableId(), expectedTableId);
+    Preconditions.checkState(tabletMeta.getLogs().isEmpty(),
```

Comment:
For the method this call this if they are fetching columns, then need to add the WALS.   Like DeleteRows calls this and is fetching columns.